nautobot/core/api/utils.py

```diff
@@ -102,9 +103,14 @@ def get_serializer_for_model(model, prefix=""):
         SerializerNotFound: if the requested serializer cannot be located.
     """
     app_name, model_name = model._meta.label.split(".")
+    if app_name == "contenttypes" and model_name == "ContentType":
+        app_name = "extras"
     # Serializers for Django's auth models are in the users app
     if app_name == "auth":
         app_name = "users"
+    # Special Case where users.Permission needs ObjectPermissionSerializer
```

Comment:
In the unittests it seems to be running as `auth.Permission` instead of `users.ObjectPermission`
<img width="1115" alt="Screen Shot 2023-04-17 at 10 29 14 AM" src="https://user-images.githubusercontent.com/46973263/232517164-698782fb-ba9f-41bc-bf5e-4852837bfce3.png">

Reply:
That's probably because of using `__all__` on the `GroupSerializer`, it's trying to pull in Django's built-in permissions since it has an M2M field on the built-in Group model. Probably the correct fix is to `exclude = ["permissions"]` on the `GroupSerializer`.

Reply:
This brings up a more general question of whether we should enhance the API generic test case to take an explicit list of the "expected" serializer fields for any given endpoint and fail if any are missing or any unexpected ones are present. Maybe a follow-on ticket?

Reply:
Solved in standup